.env.example

```
@@ -0,0 +1,67 @@
+
+# Network Configuration
+# Options: "mainnet" | "testnet"
+NEXT_PUBLIC_DEPLOYMENT=testnet
+
+# Multi-Peg Configuration
+# Configure the protocol to support different peg assets (Gold, USD, etc.)
+
+# Protocol name displayed in the navbar
+# Default: GLUON GOLD
+# Example for Gluon Dollar: GLUON DOLLAR
+NEXT_PUBLIC_PROTOCOL_NAME=GLUON GOLD
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)